src/watch.js

```diff
@@ -0,0 +1,64 @@
+import { configRead } from './config';
+
+function setTime(show) {
+
+    if (!show) {
+        const watch = document.querySelector('.webOs-watch');
+        if (watch) {
+            watch.remove();
+        }
+        return;
+    }
+
+    const watch = document.createElement('div');
+    watch.innerHTML = `<div class="webOs-watch"></div>`;
+    const style = document.createElement('style');
+
+    style.textContent =
+        `.webOs-watch {
+        position: fixed;
+        right: 0;
+        top: 0;
+        margin: 1rem 2rem;
+        background-color: rgba(0, 0, 0, 0.2);
+        border-radius: 0.5rem;
+        padding: 0.4rem;
+        font-size: 1.2rem;
+    }`;
+    document.head.appendChild(style);
+    document.body.appendChild(watch);
+
+    setInterval(() => {
+        const time = new Date();
+        let minutes = time.getMinutes().toString();
+        let houres = time.getHours().toString();
+        minutes = minutes.length < 2 ? '0' + minutes : minutes;
+        houres = houres.length < 2 ? '0' + houres : houres;
+        try {
+
+            const text = houres + ' : ' + minutes;
```

Comment:
Use [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) instead of manually building a time string.